### ISSUE TITLE
Recipe request for Plots with ribbon keyword

### ISSUE THREAD

**Issue by rafael-guerra-www** [jw3126/UnitfulRecipes.jl#65]
It would be useful to add a recipe for plotting ribbons.
They provide very nice way of displaying propagation errors during simulations using MCM.jl, for instance.
```julia
using Unitful, UnitfulRecipes, Plots
x = 1:10
plot(x, -x.^2 .* 1u"m", ribbon=5)       # OK
plot(x, -x.^2 .* 1u"m", ribbon=1u"m")   # ERROR
```

**Comment by briochemc** [comment]
I agree this would be nice to have. 

I think `ribbon` is always along the `y` axis (c.f. [my `xribbon` request](https://github.com/JuliaPlots/Plots.jl/issues/2702)), so I submitted a PR that should provide a fix dealing with just the y axis... (Waiting on CI and co.)
